lib/url.js

```diff
@@ -26,3 +26,7 @@ export const URL_REGEXP = /^((https?|ftp):\/\/)?(www.)?(((([a-z]|\d|-|\.|_|~|[\u
 
 // eslint-disable-next-line
 export const WS_REGEXP = /^(wss?:\/\/)([0-9]{1,3}(?:\.[0-9]{1,3}){3}|(?=[^\/]{1,254}(?![^\/]))(?:(?=[a-zA-Z0-9-]{1,63}\.)(?:xn--+)?[a-zA-Z0-9]+(?:-[a-zA-Z0-9]+)*\.)+[a-zA-Z]{2,63})(:([0-9]{1,5}))?$/
+
+export const IMGPROXY_URL_REGEXP = new RegExp(`^${process.env.NEXT_PUBLIC_IMGPROXY_URL}.*$`)
```

Comment:
@huumn 

make sure you add a `/` at the end of `NEXT_PUBLIC_IMGPROXY_URL`

Reply:
Are you sure? I end up with broken links: `https://imgprxy.stacker.news//v8Ghq8SIKhPhZ9mbMgCN7TKSQz5ij8pSoVqWAiwnENY/rs:fit:600:500:0/g:no/aHR0cHM6Ly9zdGF0aWMubmV3cy5iaXRjb2luLmNvbS93cC1jb250ZW50L3VwbG9hZHMvMjAyMy8wNy9iYWxhamktYmFua3MtMzgweDIxNC5qcGc`

I think this is handled in `createImageProxyUrl`

Reply:
Ah, you are right.

Then I should add a slash here so only URLs which start with `https://imgproxy.stacker.news/` are considered to be from our image proxy and not also `https://imgproxy.stacker.news.malicious.domain`

Reply:
I'll make the change. I'll grep around and make sure I don't miss somewhere.